test/versioned_workflow.go

```diff
@@ -0,0 +1,234 @@
+package test
```

Comment:
this is all exposed publicly, "test" is not a special package name or anything.  probably just needs to be moved to a _test.go file

Reply:
Good point, fixed it